src/lake/Lake/Build/Module.lean

```diff
@@ -13,6 +13,15 @@ Build function definitions for a module's builtin facets.
 
 open System
 
+
+/-- Remove adjacent duplicates. -/
+def List.squeeze [BEq α] : List α → List α
```

Comment:
@bollu Sorry, minor style quibble: Lake does not put things outside the `Lake` namespace (e.g., see `Lake.Util.Name`). Unfortunately, this does mean one cannot use dot notation with new definitions, but I think that is worth it to avoid potential clashes (e.g., with Std) when using Lake as a library.